IBAnimatable/PlaceholderDesignable.swift

```diff
@@ -18,7 +18,8 @@ public extension PlaceholderDesignable where Self: UITextField {
   var placeholderText: String? { get { return "" } set {} }
 
   public func configurePlaceholderColor() {
-    if let placeholderColor = placeholderColor, let placeholder = placeholder {
+    let text = placeholderText ?? placeholder
```

Comment:
Doesn't seem to be an issue since it was working in the both sense when I tested it 🤔